extensions/wikia/CommunityHeader/styles/index.scss

```diff
@@ -42,3 +42,9 @@
 		}
 	}
 }
+
+.wds-local-navigation-preview {
+	height: auto;
+	// this is max width of local navigation allowed
```

Comment:
It's probably defined somewhere in the JS too, right? Let's add a comment where it is.